crypto/sha/asm/sha512-x86_64.pl

```diff
@@ -2207,9 +2263,14 @@ ()
 
 	jbe	.Loop_avx2
 	lea	(%rsp),$Tbl
+# temporarily use $Tbl as index to $_rsp
+# this avoids the need to save a secondary frame pointer at -8(%rsp)
+.cfi_cfa_expression	$Tbl+`16*$SZ+3*8`,deref,+8
```

Comment:
So after this point neither -8(%rsp) nor %_rsp which is 16 * $SZ+3*8(%rsp) is reliable.
Only 16 * $SZ+3*8($Tbl) is valid in both cases.